include/SFML/Network/Ftp.h

```diff
@@ -168,7 +168,7 @@ CSFML_NETWORK_API size_t sfFtpListingResponse_getCount(const sfFtpListingRespons
 /// \brief Return a directory/file name contained in a FTP listing response
 ///
 /// \param ftpListingResponse Ftp listing response
-/// \param index              Index of the name to get (in range [0 .. getCount])
+/// \param index              Index of the name to get (in range [0 .. getCount-1])
```

Comment:
```suggestion
/// \param index              Index of the name to get (in range [0 .. getCount))
```
We can use a closing parentheses to indicate this is an open/closed set

Reply:
OK, as you like. I didn't use that mathematical notation because it is seldom used for integers.